drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -379,3 +444,5 @@ def process(transaction):
 
 		# 7. Last but not least we create the open science file format for images which is
 		# OMERO-Tiff and store it in OMERO next to the proprierary vendor format.
```

Comment:
Outdated description since the only thing that's happening here is a log_print?

Reply:
Its removed